cores/esp8266/core_esp8266_waveform_pwm.cpp

```diff
@@ -96,7 +96,7 @@ static WVFState wvfState;
 static IRAM_ATTR void timer1Interrupt();
 static bool timerRunning = false;
 
-static __attribute__((noinline)) void initTimer() {
+static void initTimer() {
```

Comment:
The noinline attribute was added here to save a few bytes of program space...

Reply:
It's in a `#pragma GCC optimize ("Os")` code section anyway. Let the compiler decide?

Reply:
Re-added the noinline attribute.